src/main/java/com/hydratereminder/command/chat/ChatMessageSender.java

```diff
@@ -0,0 +1,73 @@
+package com.hydratereminder.command.chat;
+
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.ChatMessageType;
+import net.runelite.api.Client;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import static com.hydratereminder.dictionary.HydrateWelcomeMessageDictionary.getRandomWelcomeMessage;
+
+@Slf4j
+@Singleton
+public class ChatMessageSender {
+
+    @Inject
+    private Client client;
+
+    @Inject
+    private ChatMessageTypeProvider chatMessageTypeProvider;
+
+    @Inject
+    private HydrateEmojiProvider hydrateEmojiProvider;
+
+    /**
+     * <p>Generates and sends a neatly formatted chat message prefixed by the
+     * hydrate emoji to the player
+     * </p>
+     *
+     * @param type    the type of chat message to send
+     * @param message the hydrate reminder message to display to the player
+     */
+    public void sendHydrateEmojiChatMessage(ChatMessageType type, String message) {
```

Comment:
Class `HydrateReminderPlugin` uses this method always with `ChatMessageType.GAMEMESSAGE` argument.

Can we make this method private, and expose another one like `sendHydrateEmojiChatGameMessage` that will utilize this implementation? 

The the signature of methods in `HydrateReminderPlugin` will be just:
```
chatMessageSender.sendHydrateEmojiChatGameMessage(message);
```